compiler/src/dotty/tools/dotc/util/SimpleIdentitySet.scala

```diff
@@ -7,6 +7,7 @@ import collection.mutable.ListBuffer
  */
 abstract class SimpleIdentitySet[+Elem <: AnyRef] {
   def size: Int
+  def nonEmpty: Boolean = size > 0
```

Comment:
I think it would be more idiomatic to use `isEmpty` instead. Or else neither and just use `size > 0`.